packages/react-components/src/Concepts/Icons/ReactIconGrid.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { FluentIconsProps } from '@fluentui/react-icons';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import * as ReactIcons from '@fluentui/react-icons';
+import { Input } from '@fluentui/react-input';
+import { makeStyles } from '@fluentui/react-make-styles';
+
+const useStyles = makeStyles({
+  grid: theme => ({
+    display: 'flex',
+    flexWrap: 'wrap',
+    justifyContent: 'flex-start',
+
+    'grid-list': {
```

Comment:
I think this will fix lack of proper styling for the icons:
```suggestion
    '& li': {
```